UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API/Services/ProductDataService.cs

```diff
@@ -174,5 +197,16 @@ private string ConvertLastModifiedToString(SalesCatalogueResponse salesCatalouge
             return (salesCatalougeResponse.LastModified.HasValue) ? salesCatalougeResponse.LastModified.Value.ToString(RFC1123Format) : null; 
         }
 
+        private async Task<bool> GetSalesCatalogueStorageDetails(SalesCatalogueResponse salesCatalogueResponse, string batchId)
+        {
+            essLogger.LogInformation(EventIds.SCSResponseStoreRequestStart.ToEventId(), $"SCS response store request started");
+
+            bool result = await exchangeSetStorageProvider.SaveSalesCatalogueResponse(salesCatalogueResponse, batchId);
+
+            essLogger.LogInformation(EventIds.SCSResponseStoreCompleted.ToEventId(), $"SCS response store request completed");
+
+            return result;
+        }
+
```

Comment:
Is this the best name for this method?

Should it be called SaveSalesCatalogueResponse or SaveSalesCatalogueStorageDetails which better describes the reason for the method?

Reply:
Done

Reply:
Sorry Avani I don't think I explained my thoughts very well!

I was referring to the Get (at the beginning of the method name) which infers that we are waiting to get something back while if i correctly understand the method we are really saving the Sales Catalogue response?

Reply:
Done. Aligned with you. Will take care in future.

Reply:
Thanks for doing that.

There was no issue at your end just my poor explanation!